server/game/basecard.ts

```diff
@@ -66,7 +66,7 @@ class BaseCard extends EffectSource {
     get actions(): CardAction[] {
         let actions = this.abilities.actions;
         if(this.anyEffect(EffectNames.CopyCharacter)) {
```

Comment:
Ah, now your above code makes sense.  Yeah, I think we need to fix this hack, not hack something else to make this work....

Reply:
I'll refactor it all and see if I can simplify it.